iothub/device/src/net451/IotHubClientWebSocket.cs

```diff
@@ -1,73 +1,73 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Not sure how I first found this file, but now I can't find it in the solution. There is another with the same name, but this one from the net451 folder doesn't seem to be referenced in the solution.

Reply:
We should delete this then. It's an internal class, and if don't reference it anywhere, let's delete it.